redisinsight/ui/src/pages/rdi/instance/components/header/components/rdi-config-file-action-menu/RdiConfigFileActionMenu.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import {
+  EuiPopover,
+  EuiButtonIcon,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+} from '@elastic/eui'
+import cx from 'classnames'
+import threeDots from 'uiSrc/assets/img/rdi/three_dots.svg?react'
+import uploadIcon from 'uiSrc/assets/img/rdi/upload.svg?react'
+import FetchPipelinePopover from '../fetch-pipeline-popover'
+import UploadModal from '../../../../../pipeline-management/components/upload-modal/UploadModal'
```

Comment:
to many `../` its better to use uiSrc,
also you could add index.ts to have less path